stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -69,7 +70,8 @@ internal class ChatDomainImpl internal constructor(internal val chatDomainStateF
             chatDomainStateFlow.currentUser = value
         }
 
-    override val user: LiveData<User?> = chatDomainStateFlow.user.asLiveData()
+    override val user: LiveData<User?>
+        get() = (chatDomainStateFlow as ChatDomainImpl)._userLiveData
```

Comment:
asLiveData() starts with `value` as `null` even when the `value` is already set in the stateFlow. That makes our filtering to fail because it needs the CHatDomain.user to has a value right in it's initialization.

Reply:
We can use the new ChatDomain inside the ChannelListViewModel. Then we can remove this line.